src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -244,3 +244,65 @@ def test_configure_waveform_acquisition(session):
     assert session.range == 100
     assert session.waveform_rate == 100000
     assert session.waveform_points == 400
+
+
+def test_fetch_waveform(session):
+    session.configure_waveform_acquisition(nidmm.Function.WAVEFORM_VOLTAGE, 10, 1000, 10)
+    with session.initiate():
+        number_of_points_to_read = 10
+        measurements = session.fetch_waveform(array_size=number_of_points_to_read)[1]
```

Comment:
What is the [1] for?
You dont need to call out 'array_size=number_of_points_to_read'
You can simply say 'session.fetch_waveform(number_of_points_to_read)'

Reply:
fetch_waveform returns two output waveformArray and actualNumberOfPoints respectively. len(waveformArray) should be actualNumberOfPoints. inorder to reduce the number of line assigning actualNumberOfPoints  to measurements.

will change missleading parameter name  measurements